src/Shortcode/Form/FormInstance.php

```diff
@@ -158,6 +158,7 @@ class FormInstance extends AbstractForm {
     private $isDefaultLanguage = true;
     private $languageCode = 1033;
     private $keepLabels = false;
+    private $isIgnoreNonce = false;
```

Comment:
"Ignore" is a verb. It's an imperative, not a description. So, no "is", just "do this or that".

```suggestion
    private $ignoreNonce = false;
```